tests/data/test_kdata.py

```diff
@@ -184,6 +229,11 @@ def test_KData_cuda(ismrmrd_cart):
     assert kdata_cuda.traj.kz.is_cuda
     assert kdata_cuda.traj.ky.is_cuda
     assert kdata_cuda.traj.kx.is_cuda
+    assert kdata_cuda.header.acq_info.user_int.is_cuda
```

Comment:
could we add here:
`assert kdata_cuda.header.acq_info.patient_table_position.is_cuda`
to make sure it also work for `SpatialDimension`